### ISSUE TITLE
Entering sufficient metadata for Publications Requiring Permissions Review

### ISSUE THREAD

**Issue by anaelizabethenriquez** [psu-libraries/researcher-metadata#780]
Related to #714 , when an admin works on a publication from the list of publications for which permissions data could not be found, we'd like some validation or other technical help to ensure that the admin manually entering permissions data enters data in all applicable fields.

There are four fields associated with permissions review:

- Preferred version
- Set statement
- Licence
- Embargo date

Only "Preferred version" is really necessary to move a publication into the next step in the workflow, which is version checking. My understanding is that the current logic will remove a publication from the list of publications that need a permissions check once the "Preferred version" field is no longer empty.

Since someone manually reviewing the journal's policy will find all info for all four fields, it will be more efficient to get all four fields filled out during this step.

One option would be to require all four fields not be empty in order for the publication to move off this list. The trouble with that is that some publications do not have a set statement and/or an embargo. We need a way to indicate that we've checked this even though there's no data that should get sent to ScholarSphere for that field.

Validation does happen at the time of deposit, but the current user deposit workflow does not include validation on the embargo or set statement fields (since they sometimes don't exist).

@EricDurante @ajkiessl

**Comment by EricDurante** [comment]
Thanks for writing this up, @anaelizabethenriquez!

I had to check to remember exactly how the permissions check step currently works, and it goes like this:

RMD records a timestamp indicating when it last automatically checked OA Button for permissions metadata for a publication. If that timestamp is blank, then it runs the automatic check and then updates the metadata and timestamp, but the automatic check may come up empty or with incomplete metadata. During the check, it attempts to gather the preferred version, license, set statement, and embargo end date from OA Button. For a given publication, if RMD knows that it has previously run the automatic check, but the `license` field on the publication record is still blank, then that's currently the indication that we didn't get complete permissions metadata from OA Button. It then shows publications that are in this state in the "Publications Requiring Permissions Review" list for manual admin review.

So when a publication is in this list, we know that we've already checked OA Button for permissions and got either no result or only partial metadata. With the way that things work currently, as soon as the `license` field is manually filled in by an admin, the publication will move off of this list. It looks to me like this is a spot where publications could fall through the cracks since `preferred_version` isn't required in order for a publication to move off of this list, but it will be required for the publication to show up on the final "metadata review" list.

I think that we definitely want to at least require `preferred_version` to be present for a publication to drop off of the "Publications Requiring Permissions Review" list, and perhaps all of the permissions metadata fields as you suggested. What we could do additionally is to create two more boolean flags on publications in RMD for an admin to indicate that they've done a manual check for the set statement and embargo date. Then for a publication to move off of the "Publications Requiring Permissions Review" list, it would need to either have a value for `set_statement` or the flag indicating that the manual check for set statement was done, and likewise with embargo date.

There will still be the opportunity to review/edit all of the metadata at the last step before deposit, and we can enforce the existing validations here, but we'll know that if a publication has made it that far, then the permissions metadata should be good to go.

How does that sound to everyone? @Smullz622 @ajkiessl
